run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/zsh
```

Comment:
I'm hesitant to add a script relying on `/bin/zsh`

Reply:
the Mac has moved over to this as a default, but nothing I have done should prevent bash from doing this.